app/src/main/java/com/github/swent/echo/authentication/AuthenticationService.kt

```diff
@@ -6,6 +6,9 @@ import io.github.jan.supabase.compose.auth.composable.NativeSignInResult
 /** Service to handle user authentication. */
 interface AuthenticationService {
 
+    /** Initialize the authentication service. */
+    suspend fun initialize()
```

Comment:
Why is it needed to expose an initialize function? It may be simpler to add an init block in the implementation of the authentication service.